pkg/report/report.go

```diff
@@ -115,6 +115,19 @@ func NewReporter(cfg *mgrconfig.Config) (*Reporter, error) {
 		"fatal error: runtime: cannot allocate memory",
 		"fatal error: out of memory",
 		"fatal error: newosproc",
+		"fatal error: runtime: split stack overflow",
+		"fatal error: invalid runtime symbol table",
+		"fatal error: unknown caller pc",
+		"fatal error: fault",
+		"fatal error: missing stackmap",
+		"fatal error: traceback did not unwind completely",
+		"fatal error: s.freeindex > s.nelems",
+		"fatal error: index out of range",
+		"fatal error: Connection to IP closed by remote host.",
+		"fatal error: non in-use span found with specials bit set",
+		"fatal error: runtimer: bad p",
+		"fatal error: bad sweepgen in refill",
+		"fatal error:",
```

Comment:
All of the previous ones are not needed if we add this fallback.
All of these are also details of a particular Go version and can change in future.

Reply:
Ah oops, ofc, somehow I missed these were regexps! :)